packages/app/contexts/ModalContext.tsx

```diff
@@ -9,6 +9,7 @@ import {
 } from "react";
 
 export enum ModalId {
+  WAITING_TRANSACTION = "waiting_transaction",
```

Comment:
Knowing here that's the NFT related transaction is good.

```suggestion
  NFT_WAITING_TRANSACTION = "nft_waiting_transaction",
```

Reply:
I think we can have a default waiting transaction state. What do you think?

Reply:
We currently have: `STACK_APPROVE_PROCESSING`, `CANCEL_STACK_PROCESSING`, `STACK_CREATION_PROCESSING`

to be consistent would be:`NFT_MINT_PROCESSING`

I don't know if using the `TRANSACTION_PROCESSING` or `WAITING_TRANSACTION` in all of them is better. 
But I'll let you decide on that.

Reply:
Will change to something more specific.